packages/core/strapi/tests/api/basic-dp-compo.test.e2e.js

```diff
@@ -161,6 +161,13 @@ describe('Core API - Basic + compo + draftAndPublish', () => {
     data.productsWithCompoAndDP.shift();
   });
 
+  describe('database state', () => {
+    test('components have been removed from the database', async () => {
+      const dbComponents = await strapi.db.query('default.compo').findMany({});
```

Comment:
Let's see what the fun test run says but it may not work as the test database may have data left from the other tests

Reply:
Isn't the database pruned in each test file? 
Maybe I can add a query in the findMany referring to the component added in this test file then.

Reply:
I'm not sure, it would be nice to check and know. Js wrote the code

Reply:
I will add the query just in case then.